ci/acceptance/main.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"database/sql"
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/lib/pq"
+
+	"github.com/cloudfoundry-community/go-cfenv"
+)
+
+var creds map[string]interface{}
+
+func main() {
+	env, _ := cfenv.Current()
+	services, _ := env.Services.WithLabel(os.Getenv("SERVICE_NAME"))
+	if len(services) != 1 {
+		log.Fatalf("Expected one service instance; got %d", len(services))
+	}
+	creds = services[0].Credentials
+
+	http.HandleFunc("/mysql", mysqlHandler)
+	http.HandleFunc("/postgres", postgresHandler)
+	http.ListenAndServe(":"+os.Getenv("PORT"), nil)
+}
+
+func mysqlHandler(w http.ResponseWriter, r *http.Request) {
+	u, _ := url.Parse(creds["uri"].(string))
+	u.Host = fmt.Sprintf("tcp(%s)", u.Host)
+	u.RawQuery = ""
+	dsn := strings.Replace(u.String(), "mysql://", "", -1)
+
+	db, err := sql.Open(u.Scheme, dsn)
+	if err != nil {
+		log.Println(err.Error())
```

Comment:
could wrap these two lines (the log and header writing) in a `handleError(string)` function